manifests/apps/jupyter/jupyter-app.yaml

```diff
@@ -164,7 +164,7 @@ spec:
     3.35641 0.286707 3.70796 0.553234 3.96186 0.902636C 4.21577 1.25204 4.3607
     1.66872 4.37842 2.10027C 4.39529 2.6838 4.18131 3.25044 3.78293 3.67715C
     3.38455 4.10387 2.83392 4.35623 2.25061 4.37943Z"/>\n</defs>\n</svg>
-  getStartedLink: 'https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html'
+  getStartedLink: 'https://opendatahub.io/docs/getting-started-with-open-data-hub/#launching-jupyter-and-starting-a-notebook-server_get-started'
```

Comment:
this link is good, thank you very much. In general, as mentioned, I think we need to be careful not to mix access.redhat links to RHOSDS in the files here elsewhere. Question is whether we find the alternatives at opendatahub.io.